src/main/java/de/komoot/photon/PhotonDoc.java

```diff
@@ -81,8 +81,6 @@ public boolean isUsefulForIndex() {
 
 		if(name.isEmpty()) return false;
 
-		if(linkedPlaceId > 0) return false;
-
-		return true;
+		return linkedPlaceId <= 0;
```

Comment:
Yeah I agree. Sometimes IntelliJ helps me with some proper structure, but in this case it indeed makes it harder to understand.